recipes/libiconv/all/conanfile.py

```diff
@@ -44,9 +44,8 @@ def build_requirements(self):
             self.build_requires("msys2/20200517")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        archive_name = "{0}-{1}".format(self.name, self.version)
-        os.rename(archive_name, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+        tools.rename(archive_name, self._source_subfolder)
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
```

Small review error. GitHub isn't perfect yet with multi-line suggestions